rabbitmq/check.py

```diff
@@ -353,7 +353,7 @@ def get_connections_stat(self, instance, base_url, object_type, vhosts, custom_t
         for conn in data:
             if conn['vhost'] in vhosts:
                 stats[conn['vhost']] += 1
-                connection_states[conn['state']] += 1
+                connection_states[conn.get('state', 'direct')] += 1
```

Comment:
Could you add a quick comment explaining why we're defaulting to `direct` ?

Reply:
I decided to default to `direct` instead of `unknown` to account for the number of direct type connections where the state doesn't really apply. I believe this will avoid the potential question from clients asking "what are these connection_state:unknowns?" and us replying back with "those are actually direct connections".

Reply:
Sorry, I meant could you add a comment in the code.

Reply:
Done